plugin.go

```diff
@@ -145,3 +153,9 @@ func getPlugin(cp *C.CPlugin) *Plugin {
 	defer plugins.RUnlock()
 	return plugins.mapping[uintptr(unsafe.Pointer(cp))]
 }
+
+func deletePlugin(cp *C.CPlugin) {
```

Comment:
Not really sure if this function is needed, I believe that there'll be just one place where it'll be called (in `dispatchPluginBridge` function)